service/collector/cluster_info.go

```diff
@@ -0,0 +1,50 @@
+package collector
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	clusterLabel = "cluster_id"
+	statusLabel  = "status"
+)
+
+var (
+	clustersDesc *prometheus.Desc = prometheus.NewDesc(
+		prometheus.BuildFQName(Namespace, "", "cluster_info"),
+		"Cluster information.",
+		[]string{
+			clusterLabel,
+			statusLabel,
+		},
+		nil,
+	)
+)
+
+func (c *Collector) collectClusterInfo(ch chan<- prometheus.Metric) {
+	c.logger.Log("level", "debug", "message", "collecting metrics for clusters")
+
+	opts := v1.ListOptions{}
+	clusters, _ := c.g8sClient.ProviderV1alpha1().AWSConfigs("").List(opts)
+
+	for _, cluster := range clusters.Items {
```

Comment:
Should there be paging for listing? Here is an example I've implemented: https://github.com/giantswarm/cluster-operator/blob/master/pkg/v3/resource/certconfig/current.go#L37-L63

Reply:
yeah you are right. I will do